### ISSUE TITLE
Switch to core-js 3.x not mentioned in 2.3.1 release changelog

### ISSUE THREAD

**Issue by aaarichter** [DanielSchaffer/webpack-babel-multi-target-plugin#49]
Hi,

The change to bump core-js 3.x in 2.3.1 is kind of a breaking change for me due to the fact that one has to exclude the core-js node module. see https://github.com/zloirock/core-js/issues/514

In my case I have core-js pinned down to 2.x, which will cause the following error

```
ERROR in ./core/frontend/test.js?babel-target=legacy
Module not found: Error: Can't resolve 'core-js/modules/es.array.iterator?babel-target=legacy' 
 @ ./core/frontend/test.js?babel-target=legacy 5:0-44
```

maybe you should bump a major or mention the dependency to core-js 3.x

**Comment by DanielSchaffer** [comment]
Oh heck, I'm sorry about that! I wasn't planning on including that until 2.4, I'm not sure how that snuck in.

I'll see if I can revert that change and publishing a patch version this weekend. I've also updated the [release notes for 2.3.1](https://github.com/DanielSchaffer/webpack-babel-multi-target-plugin/releases/tag/v2.3.1) to reflect the change.

**Comment by aaarichter** [comment]
No problem @DanielSchaffer . Thanks for addressing it 😺

**Comment by DanielSchaffer** [comment]
@aaarichter try `2.3.3-next.1`. That should include everything from `2.3.1` except the core-js bump.

Note that I had to skip from `2.3.1` to `2.3.3` for this because I'd already started a `next` release for `2.3.2` - so it had to be `2.3.3` to avoid npm package version conflicts for prereleases.

**Comment by DanielSchaffer** [comment]
This is now released in [2.3.3](https://github.com/DanielSchaffer/webpack-babel-multi-target-plugin/releases/tag/v2.3.3)
